csvcubed-pmd/csvcubedpmd/urireplacement/urireplacement.py

```diff
@@ -0,0 +1,90 @@
+import click
+import sys
+import os
+
+from chardet.universaldetector import UniversalDetector
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+
+def _file_in_line(line: str) -> bool:
+    """
+    Returns true if line has file:/ in it
+    """
+    return "file:/" in line
+
+
+def _line_replace(line: str, values: tuple[tuple[str, str]]) -> str:
```

Comment:
I'm not sure `tuple[tuple[str, str]]` is a valid type. I suspect you mean `Tuple[str, str]` where `Tuple` is imported from the `typing` module.